### ISSUE TITLE
Molarity 1.4.0 crashes in the State wrapper when opening the About dialog

### ISSUE THREAD

**Issue by samreid** [phetsims/molarity#48]
See https://github.com/phetsims/sun/issues/374

You can test this in https://phet-io.colorado.edu/sims/molarity/1.4.0/wrappers/state/state.html by going to the state wrapper and opening the about dialog and waiting 1 second.  In 1.4.0 this results in a crash.

We have 2 commits we would like to cherry pick for a maintenance release.

@pixelzoom would you like to tackle this?  The commits to cherry pick are:
https://github.com/phetsims/sun/commit/76ce03cafb41597f4fa775a2fb4d55aea2b425db
https://github.com/phetsims/sun/commit/85fbf335bf1301ae853c23db28a1852eea4a9795

These cherry-picks are isolated and don't need further testing than making sure the state wrapper doesn't crash when opening the about dialog and waiting 1 second.

Also tagging QA so they are aware of this bug, thanks! @KatieWoe @JRomero0613

**Comment by pixelzoom** [comment]
Patched by doing this:

```
grunt checkout-target --repo=molarity --target=1.4
cd sun
git checkout -b molarity-1.4
git checkout master -- js/buttons/ButtonModel.js
git add js/buttons/ButtonModel.js
git commit -m “https://github.com/phetsims/molarity/issues/48”
git push origin molarity-1.4
cd molarity
grunt --brands=phet,phet-io
cp build/phet/dependences.json dependences.json 
git add dependences.json 
git commit -m “https://github.com/phetsims/molarity/issues/48”
git push
```

Then I pointed Chrome at molarity/build/phet-io/ and opened the State wrapper.  After opening the About dialog, I experienced no crash.  (The About dialog did not open in the bottom sim instance, but @samreid said that is as desired.)

**Comment by pixelzoom** [comment]
@samreid Do we need an RC for this? Or do you want to just confirm in a local build?

**Comment by samreid** [comment]
I ran:

```
grunt checkout-target --repo=molarity --target=1.4
cd molarity
grunt --brands=phet,phet-io
```

I tested locally for about 3 minutes, testing the about dialog in the state wrapper and tapping through the normal sim.  I did not see anything out of the ordinary.  I would love if there was a trivial way to compare code changes for maintenance releases, but I do not know how to do that at the moment.  If the only code change is the ButtonModel 2 commits, this should be good to go.

**Comment by pixelzoom** [comment]
`grunt production` fails with "Error: Aborted production deployment: It appears that the last deployment was for production." Apparently it's impossible to publish a maintenance release without publishing an RC version first.  So I’m going to follow the full process for a maintenance release - publish an RC, create a GitHub issue to have it tested by QA, etc.

**Comment by pixelzoom** [comment]
Testing in 1.4.1-rc.1, https://github.com/phetsims/QA/issues/170.

**Comment by KatieWoe** [comment]
Testing this right now.  Not seeing any crashes with the about dialog, but I am noticing that the dialog doesn't then open up on the bottom screen.  For phetsims/QA#170

**Comment by pixelzoom** [comment]
As noted in https://github.com/phetsims/molarity/issues/48#issuecomment-413293879:
> (The About dialog did not open in the bottom sim instance, but @samreid said that is as desired.)

**Comment by pixelzoom** [comment]
More detailed description of why the About dialog doesn't appear in the bottom sim instance, from Slack:

Chris Malley [12:31 PM]
I’m testing the patch to https://github.com/phetsims/molarity/issues/48.  In the State wrapper, I don’t see a crash when opening the About dialog.  But the About dialog does not open in the bottom instance of the sim. (edited)


Sam Reid [12:32 PM]
Right, we decided that the about dialog was “transient” and hence we implemented it with `phetioState: false` so it wouldn’t be recorded or restored as part of the state.
